explainaboard/metric.py

```diff
@@ -151,7 +152,13 @@ def aggregate_stats(self, stats: MetricStats) -> np.ndarray:
         :param stats: stats for every example
         :return: aggregated stats
         """
-        return np.mean(stats.get_data(), axis=0)
+
+        res = copy.deepcopy(stats)
```

Comment:
We don't need a deepcopy here, do we? Couldn't we just do `data = get_data()` and then use `data` for the rest of the function? I'd like to avoid copying memory unnecessarily.

Reply:
no probolem and done